### PR TITLE
Adding ErrorBar test, fixing storybook controls

### PR DESCRIPTION
**Pull request by PavelVanecek**
## Description

Trying to understand error bars, and vertical layouts.

## Related Issue

https://github.com/recharts/recharts/issues/4583

### CODE REVIEW

**Review by ckifer — APPROVED** [review]

File: test/cartesian/ErrorBar.spec.tsx [hunk 1]
```diff
@@ -215,4 +217,170 @@ describe('<ErrorBar />', () => {
...
+        },
+        {
+          textContent: '3600',
+          x: '57',
+          y: '5',
+        },
+      ]);
+    });
+
+    test('ErrorBar not extend XAxis domain - this looks like a bug to me!', () => {
```
Comment (PavelVanecek) [inline test/cartesian/ErrorBar.spec.tsx:310]: Interesting, I found https://github.com/recharts/recharts/issues/634 which has a screenshot of the exact same problem and says it's fixed. I will backport the same test to 2.x branch to see where it has regressed.

Comment (ckifer) [reply test/cartesian/ErrorBar.spec.tsx:310]: Hmmmmm good catch

Comment (PavelVanecek) [reply test/cartesian/ErrorBar.spec.tsx:310]: Okay the exact same problem is on master: 
image

Comment (ckifer) [reply test/cartesian/ErrorBar.spec.tsx:310]: Good to know

**Pull request merged**